RecoTauTag/RecoTau/src/PositionAtECalEntranceComputer.cc

```diff
@@ -16,7 +16,8 @@ void PositionAtECalEntranceComputer::beginEvent(const edm::EventSetup& es) {
   bField_z_ = bField->inTesla(GlobalPoint(0., 0., 0.)).z();
 }
 
-reco::Candidate::Point PositionAtECalEntranceComputer::operator()(const reco::Candidate* particle, bool& success) const {
+reco::Candidate::Point PositionAtECalEntranceComputer::operator()(const reco::Candidate* particle,
+                                                                  bool& success) const {
```

Comment:
this is not a part of the listed PRs used for backport.
Apparently, the change is coming from 
https://github.com/cms-sw/cmssw/pull/30571/files#diff-225356b5230d327db61da9f1a9a6514e03ec42126c806f2a46fc54c97a7a64ebL19-R20

it seems acceptable to include though.

Reply:
I think it is rather this commit https://github.com/cms-sw/cmssw/commit/9457485ec9f806bc1f877d8a28bbbc2d8d87f50f which is probably not backported but where I applied the code-format at some point. Sorry for the confusion!